### PR TITLE
Server Profile

### PR DESCRIPTION
**Pull request by tmiotto**
### Description
Implements Server Profile resource

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [x] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by tmiotto** [comment]
Travis failing due to fauxhai lib not found in ruby 2.1

File: spec/fixtures/cookbooks/oneview_test/recipes/server_profile_properties.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: oneview_test
+# Recipe:: server_profile_delete
```
Comment (hdiomede) [inline spec/fixtures/cookbooks/oneview_test/recipes/server_profile_properties.rb]: Wrong recipe name

**Review by hdiomede — CHANGES_REQUESTED** [review]

**Review by hdiomede — APPROVED** [review]

**Comment by jsmartt** [comment]
Fauxhai's latest release introduces yet another dependency on a minimum Ruby version. We can either add it to our Gemfile and specify the version or come up with another, more sane way of managing these versions across our cookbooks. Thoughts?

**Comment by tmiotto** [comment]
Opened issue #85 

We can came up with a solution there.

**Pull request merged**